fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalTopNOperator.java

```diff
@@ -23,45 +24,55 @@ public class LogicalTopNOperator extends LogicalOperator {
     private final List<Ordering> orderByElements;
     private final long offset;
     private final SortPhase sortPhase;
+    private final TopNType topNType;
     private boolean isSplit = false;
 
     public LogicalTopNOperator(List<Ordering> orderByElements) {
-        this(DEFAULT_LIMIT, null, null, null, DEFAULT_LIMIT, orderByElements, DEFAULT_OFFSET, SortPhase.FINAL, false);
+        this(DEFAULT_LIMIT, null, null, null, DEFAULT_LIMIT, orderByElements, DEFAULT_OFFSET, SortPhase.FINAL,
+                TopNType.ROW_NUMBER,
```

Comment:
Why the default topn type is row number?

Reply:
The ordinary topn is limit by specified number of rows, which matches the meaning of window function `row_number`, so I name it as `ROW_NUMBER`